pages/apis.md

```diff
@@ -12,6 +12,20 @@ Generate an [access token](https://buildkite.com/user/api-access-tokens).
 
 All webhooks contain an [`X-Buildkite-Token` header](/docs/apis/webhooks#http-headers) which allows you to verify the authenticity of the request.
 
+## API security
+
+We are a member of the [GitHub Secret Scanning Program](https://docs.github.com/en/code-security/secret-scanning/secret-scanning-partnership-program/secret-scanning-partner-program).
```

Comment:
```suggestion
Buildkite is a member of the [GitHub Secret Scanning Program](https://docs.github.com/en/code-security/secret-scanning/secret-scanning-partnership-program/secret-scanning-partner-program).
```

Reply:
See the section on [Formality](https://github.com/buildkite/docs/blob/main/styleguides/writing-style.md#formality) in the Writing style guide.